src/internal/components/TextInput.tsx

```diff
@@ -49,6 +51,7 @@ export function TextInput({
 
   return (
     <input
+      style={style}
```

Comment:
Why do we need to add an inline style attribute here? We should try to stick to `className` for consistency